### DISCUSSION TITLE
Hide pricing information from certain groups?

### DISCUSSION THREAD

**Discussion by anonkwe** [inventree/InvenTree#4466] [Q&A]
Is it possible to mask pricing from a group that only has to manage in house stock inventory? Pricing is displayed even if a group doesnt have access to po's and also on the mobile app..

![image](https://user-images.githubusercontent.com/61838766/223684385-f4a91b41-9d1c-4402-9d55-2a4f22dc4443.jpeg)

**Comment by SchrodingersGat** [comment]
This is related to https://github.com/inventree/InvenTree/issues/4022 - currently there is no permissions group for pricing, although it does make sense!

**Reply by SchrodingersGat** [reply]
Also this should be raised against the [inventree-app](https://github.com/inventree/inventree-app/) repo

**Reply by anonkwe** [reply]
I don't think discussion's enabled on the repo

**Reply by matmair** [reply]
Please file problems/questions with/about a part of the ecosystem in the respective GitHub repo (app in inventree-app, website in inventree-website, ...). If discussions are not enabled, file an issue.

**Reply by anonkwe** [reply]
anything? this is quite some critical basic options that should be tought of for stocktakers...

**Reply by SchrodingersGat** [reply]
This would require #4022 to be implemented. If you would like to see this implemented sooner, you can sponsor the feature. Otherwise, it will need one of the devs to assign their own free time to it.

**Reply by anonkwe** [reply]
We really would love to encourage you guys to do get this fundamental option up and going asap, just let me know how much hours do you estimate this to get done, I'll give a proper patreon no problem.  

Also, please don't get me wrong when I ask for follow ups, I know this is open source and you guys put personnal time into this great thing. I don't intend to be rude, it's just that I'm confused and feel that I'm left on seen when I  try to inform about issues or missing options that we, as one of the biggest manufacture of stairtreads and components in North America, notice. 

Getting no response at all or being answered to repost somewhere else, sometimes for the third time, compared to discussions on how to bulk import stuff through a cvs file getting over 40 replies is a bit confusing to be honest... 

As far as we are concerned, it's been a while that oodoo and  partsbox are out of the equation for many valuable reasons including mobile app.  

Believe me when I say that I do understand and appreciate the amount of free/personal time you guys are giving to your project, I'm well placed to understand. But I also think that having manufactures across the globe adopting, modeling and using your solution should be a motivation in some way!

Anyways, just let me know how much hours it should take for such additions and we'll gladly contribute to the project in accordance of your estimate, that's for sure!

**Reply by anonkwe** [reply]
@SchrodingersGat @matmair

**Reply by SchrodingersGat** [reply]
@anonkwe if you would like to discuss sponsoring development of the required features to get this working in the app, please reach out to me at support@inventree.org 

Regarding replies to discussions - the developers (including myself) all work demanding full time jobs, and have to pick and choose which issues we devote our precious available time to! Unless we're getting paid, that's all you can reasonably expect ;)

**Reply by SchrodingersGat** [reply]
@anonkwe 

> Anyways, just let me know how much hours it should take for such additions and we'll gladly contribute to the project in accordance of your estimate, that's for sure!

I'm assuming that as you never reached out, you're no longer interested in pursuing this?

**Reply by anonkwe** [reply]
@SchrodingersGat    it was critical for the comapnion, but i'll be able to modulate this through studio

**Reply by anonkwe** [reply]
@SchrodingersGat i'll tip you that's for sure, but right now, making work the scanner is priority uno
